### ISSUE TITLE
undefined reference to `BBARRAYDATAINDEX'

### ISSUE THREAD

**Issue by olejr** [bmx-ng/bcc#95]
```
SuperStrict

Global arr:Int[] = [1, 2, 3, 4, 5]
Global id:Int

Function ArrayTest()
	id = arr[3]  'Error on this line
End Function

ArrayTest()
```
Fail to compile in debug mode. 
bcc v0.54 (Did a pull & build now, just to make sure I had the latest)

**Comment by woollybah** [comment]
It sounds like you have a file in your brl/pub modules that hasn't been rebuilt properly.
Without a "framework", *all* brl and pub modules are automatically imported into your app.

For example, if you add ```Framework brl.standardio``` does the problem go away?

**Comment by olejr** [comment]
Was about to say no.
But then, why not try to delete the mod folder, and do a fresh pull of everything.
Did a ```bmk makemods -g x86``` and ```bmk makemods -h -g x86```.
Ended up with a some  unknown modules.
Did a ```bmk makemods -g x86``` and ```bmk makemods -h -g x86``` again.

And now it compiles..
Why I had to build modules x2 I don't know..
Doing this on Win8 with tdm64 4.8.x btw..
(Only thing I can't get working now is "selfcompiling" bmk under NG.
bombs out with a ```error: conflicting types for 'GetSystemInfo'```)
